git_machete/client.py

```diff
@@ -2096,8 +2097,8 @@ def create_github_pr(
                 opt_yes=False)
 
         current_user: Optional[str] = git_machete.github.derive_current_user_login()
-        debug(f'create_github_pr({head})', f'organization is {org}, repository is {repo}')
-        debug(f'create_github_pr({head})', 'current GitHub user is ' + (current_user or '<none>'))
+        debug(f"{inspect.stack()[0].function}({head})", f'organization is {org}, repository is {repo}')
+        debug(f"{inspect.stack()[0].function}({head})", 'current GitHub user is ' + (current_user or '<none>'))
```

Comment:
(at least move method name extraction to `debug`... and if possible, also the params!)